src/auth/guards/two-factor-auth.guard.ts

```diff
@@ -0,0 +1,5 @@
+import { Injectable } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+
+@Injectable()
+export class TwoFactorAuthGuard extends AuthGuard('2FA.bearer') {}
```

Comment:
hmm, to be honest I don't know if there are any naming conventions for auth guards, but I guess I would change it to either: `2fa.bearer` or `2fa-bearer`